fintrack/choices.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class TransactionCategoriesChoices(models.TextChoices):
+    Withdrawl = "Withdrawl", _("Withdrawl")
```

Comment:
```suggestion
    withdraw = "WITHDRAW", "Withdraw"
```

Reply:
are we supporting multiple languages or doing translation in other places if not then remove gettext support